backend/src/main/java/de/neuefische/backend/controller/UserController.java

```diff
@@ -22,12 +23,13 @@ public List<User> listUsers(){
     }
 
     @PostMapping()
-    public User addUser(@RequestBody User user){
-        return userService.addUser(user);
+    public User addUser(@RequestBody UserRequest userRequest){
+        User saveUser = new User(userRequest.id(), userRequest.name(), userRequest.visitedCountries());
```

Comment:
One might to try name the variable `userToBeSaved` since it creates a nice sentence in combination with `userService.addUser(userToBeSaved);`, but this is fine for now!